src/orders/src/main/java/mushop/orders/resources/PaymentRequest.java

```diff
@@ -65,4 +67,20 @@ public float getAmount() {
     public void setAmount(float amount) {
         this.amount = amount;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
no need to check for `o == null` here, because you already check for it above and return if it's true.

Reply:
line 73 checks for reference equality - not `null`.